skins/shared/scripts/onScroll.js

```diff
@@ -45,7 +45,7 @@ define('wikia.onScroll', ['jquery', 'wikia.window'], function ($, window) {
 	function init() {
 		window.addEventListener(
 			'scroll',
-			$.debounce(debounceRate, true, trigger)
+			$.throttle(delay, trigger)
```

Comment:
The only thing that uses it is BannerNotification and I checked it - it works correctly.